Issue1〜3の修正

Riokyanpu opened the pull request:
Issue1〜3の解消

Riokyanpu committed:
Issue1〜3の修正

Riokyanpu commented:
Issue1~3の内容をsecond_trainingでCreate Pull Requestしてしまいました。
ご対応よろしくお願い致します。

OhyaYouki commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,9 +18,10 @@ def plan_params
-  def getWeek
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
-
+  def get_Week
```
`get_Week`の修正ですが、こちらはメソッド名ですので
`大文字のキャメルケース`から`小文字のスネークケース`に変更する必要がございます！

現状ですと、アンダーバーの後の`Week`が未修正となっておりますので
改めてカリキュラムを見直して`小文字のスネークケース`に修正をお願いいたします。

またコントローラーに修正すべきgetWeekは2箇所ございますので合わせて修正をお願いいたします。

OhyaYouki commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,9 +18,10 @@ def plan_params
-
+  def get_Week
+    # 役割の内容を記述することが重要（week_days_labels:曜日のラベルが格納されていることを示す）
+    week_days_labels = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
```
```ruby
week_days_labels = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
```
上記の修正に関して

```
# 役割の内容を記述することが重要（week_days_labels:曜日のラベルが格納されていることを示す）
```
こちらのコメントアウトの内容はまさしくおっしゃる通りなのですが、変数名の修正はこのカリキュラムでは行わないようにお願いいたします。

今後の実装やカリキュラムでの案内と食い違いが発生してしまい混乱を招く恐れがあるためです。

`week_days_labels`から`wdays`と元に戻しておきましょう。

OhyaYouki commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +35,8 @@ def getWeek
       end
-      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans}
+      # 変数名の役割がわかる内容で示すことが重要(days→das_info:１日分の情報を格納している。)
+      das_info = { month:  (@todays_date + x).month, date:  (@todays_date+x).day, plans:  today_plans}
```
```ruby
das_info = { month: (@todays_date + x).month, date: (@todays_date+x).day, plans: today_plans}
```
`das_info`から`days`にもとに戻しておきましょう。

OhyaYouki commented on app/views/calendars/index.html.erb:
```diff
@@ -1,8 +1,8 @@
-<div>
+<div class="form">
```
ビューの修正は`不要なdivタグの削除`になります。

```ruby
<div class="form">
```
上記は元々は`<div>`のようにclass名のないdivタグでした。
このようにclassのないdivタグは不要な要素となりますので削除が可能になります。

現状のビューにも不要なdivタグがございますのでそちらの削除をお願いいたします。

OhyaYouki commented on app/views/calendars/index.html.erb:
```diff
@@ -1,8 +1,8 @@
+    <div class="label">日付を選択</div>
     <%= f.date_field :date, class: 'date-select' %>
-    <%= f.label :予定を入力, class: 'label' %>
+   <div class="label">予定を入力</div>
```
```ruby
   <div class="label">日付を選択</div>
   <%= f.date_field :date, class: 'date-select' %>
   <div class="label">予定を入力</div>
```
上記は修正の必要なコードではないため以下のうように元に戻しておきましょう。
```ruby
    <%= f.label :日付を選択, class: 'label' %>
    <%= f.date_field :date, class: 'date-select' %>
    <%= f.label :予定を入力, class: 'label' %>
```

OhyaYouki commented on app/views/calendars/index.html.erb:
```diff
@@ -11,11 +11,9 @@
-        <% if day[:plans].length != 0 %>
+        <div class='date'><%= "#{day[:month]}/{#day[:date]}"%></div>
+           <ul class='content'>
+        <% if day[:plans].present? %>
```
```ruby
        <div class='date'><%= "#{day[:month]}/{#day[:date]}"%></div>
           <ul class='content'>
        <% if day[:plans].present? %>
```
上記も修正の必要はございません。以下のように元に戻しておきましょう。
```ruby
        <div class='date'>
          <% day[:month] %>/<% day[:date] %>
        </div>
        <ul class='content'>
        <% if day[:plans].length != 0 %>
```

OhyaYouki reviewed (commented):
レビュー依頼ありがとうございます。

コメントいたしましたので、ご確認をお願いいたします。

コードの修正後、コミット・プッシュを行った上で再度レビュー依頼をお送りください。
ご対応のほど、よろしくお願いいたします。

Riokyanpu committed:
2回目　Issue1~3の修正

Riokyanpu committed:
Issue 1~3修正　2回目

controller.rb
get_week 1箇所（５行目）
wdayからwdaysに変更（２３行目）

index.html.erb
<div>(10 行目削除)

minachanchan commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,9 +18,10 @@ def plan_params
   end
 
-  def getWeek
+  def get_Week
```
```ruby
def get_Week
```
こちらスネークケースに変更できていていいですね！
メソッド名の命名規則は小文字のスネークケースですので「W」も小文字に修正しましょう！
合わせて呼び出しの5行目の記述も変更しましょう。

minachanchan commented on app/views/calendars/index.html.erb:
```diff
@@ -7,7 +7,6 @@
     <%= f.submit '保存' %>
   <% end %>
 </div>
```
不要なdivタグが削除できていていいですね！
今回こちらのビューファイルには不要なdivタグが二組ございます。
開始タグ、閉じタグ含め残りの不要なdivタグについても削除しましょう！

minachanchan reviewed (commented):
実装お疲れ様です！レビューのご依頼ありがとうございます。
着実に完成に近づいておりますね！

コメントいたしましたので、ご確認頂けますと幸いでございます。


実装完了まで引き続き、一緒に頑張りましょう！
修正の後、再度レビューのご依頼頂けますようお願い申し上げます。

Riokyanpu committed:
3回目修正

get_weekに変更
divタグ２箇所削除

Riokyanpu commented:
確認お願いします。

koyamatakumi commented on app/views/calendars/index.html.erb:
```diff
@@ -6,8 +6,7 @@
   <% end %>
-</div>
-<div>
+
```
不要なdiv要素が現在も残っているため修正しましょう。

koyamatakumi reviewed (commented):
Riokyanpu様
レビュー依頼フォームの送信ありがとうございます。 修正いただいた内容を確認させていただき、追加でコメントいたしましたのでご確認お願いいたします。
修正後はコミット、プッシュを忘れずに行い、改めてレビュー依頼フォームの送信をお願い致します。着実にゴールに近づいています。ラストスパート頑張っていきましょう！

Riokyanpu committed:
4回目

Riokyanpu commented:
確認お願いします。

TakashiKozuma commented:
レビュー依頼ありがとうございます。
LGTMです！issueの解決お疲れさまでした。

カリキュラムに従い、次にお進みください！！
